docs/topics/pagination.txt

```diff
@@ -182,6 +196,17 @@ Methods
     Returns a :class:`Page` object with the given 1-based index. Raises
     :exc:`InvalidPage` if the given page number doesn't exist.
 
+.. method:: Paginator.page_force_into_range(number)
+
+    .. versionadded:: 2.0
+
+    Returns a :class:`Page` object with the given 1-based index.  If the page
```

Comment:
Please use "Return" instead of "Return_s_" and don't use double whitespaces after periods.